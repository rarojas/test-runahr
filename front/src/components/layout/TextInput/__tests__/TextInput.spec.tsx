import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import { TextInput } from '../index';

const defaultProps = {
    input: { value: 'value' },
    label: 'label',
    type: 'text',
    meta: { touched: false, error: false }
  },
  TestTextInput = (props: any) => <TextInput {...defaultProps} {...props} />;

describe('Render TextInput', () => {
  it('render correctly component', () => {
    const TextInputComponent = renderer.create(<TestTextInput />).toJSON();
    expect(TextInputComponent).toMatchSnapshot();
  });
});
