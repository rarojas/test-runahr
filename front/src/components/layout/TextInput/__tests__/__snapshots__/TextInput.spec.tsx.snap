// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render DateInput render correctly date component 1`] = `
Array [
  <small
    className="sc-jWBwVP jvELBg"
  >
    label
  </small>,
  <input
    className="sc-hMqMXs qIUiT"
    placeholder="label"
    type="text"
    value="value"
  />,
]
`;

exports[`Render TextInput render correctly component 1`] = `
Array [
  <small
    className="sc-jWBwVP jvELBg"
  >
    label
  </small>,
  <input
    className="sc-hMqMXs qIUiT"
    placeholder="label"
    type="text"
    value="value"
  />,
]
`;
